<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConvFitAddWorkspaceDialog</class>
 <widget class="QDialog" name="ConvFitAddWorkspaceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>181</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Indirect Data Selection</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="gbSelectData">
     <property name="title">
      <string>Select data</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="loInput">
        <item row="0" column="0">
         <widget class="QLabel" name="lbWorkspace">
          <property name="text">
           <string>Workspace</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lbResolution">
          <property name="text">
           <string>Resolution</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QCheckBox" name="ckAllSpectra">
          <property name="text">
           <string>All Spectra</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="MantidQt::MantidWidgets::DataSelector" name="dsWorkspace" native="true">
          <property name="ShowGroups" stdset="0">
           <bool>false</bool>
          </property>
          <property name="showLoad" stdset="0">
           <bool>false</bool>
          </property>
          <property name="autoLoad" stdset="0">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="MantidQt::MantidWidgets::DataSelector" name="dsResolution" native="true">
          <property name="ShowGroups" stdset="0">
           <bool>false</bool>
          </property>
          <property name="showLoad" stdset="0">
           <bool>false</bool>
          </property>
          <property name="autoLoad" stdset="0">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="lbWorkspaceIndices">
          <property name="text">
           <string>Workspace Indices</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="leWorkspaceIndices"/>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="spacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbAdd">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbClose">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MantidQt::MantidWidgets::DataSelector</class>
   <extends>QWidget</extends>
   <header>MantidQtWidgets/Common/DataSelector.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>addData()</slot>
  <slot>closeDialog()</slot>
 </slots>
</ui>
